static/app/views/insights/common/components/widgets/types.tsx

```diff
@@ -0,0 +1,25 @@
+import type {PageFilters} from 'sentry/types/core';
+
+/**
+ * These props are common across components that are required to dynamically
+ * render an Insight Chart Widget
+ */
+export interface LoadableChartWidgetProps {
+  // TODO(billy): This should be required when all chart widgets are converted
+  /**
+   * Unique ID for the widget
+   */
```

Comment:
That makes sense, but if that's the case, `TimeSeriesWidgetVisualization` should use `Partial<LoadableChartWidgetProps>`. ID shouldn't be required there, just for the Insights chart